src/main/java/ZadaniaWzorceProjektowe_10_09/Zadanie2/ChangedWheelsCar.java

```diff
@@ -0,0 +1,18 @@
+package ZadaniaWzorceProjektowe_10_09.Zadanie2;
+
+public class ChangedWheelsCar extends CarDecorator {
+
+    int wheelsDimensions;
+
+    public ChangedWheelsCar(Car car, int wheelsDimensions) {
```

Comment:
Może fajnie by było stworzyć enum reprezentujący rozmiar pól? Bo koła zazwyczaj mają określony rozmiar,i to by od razu zabezpieczyło innych developerów przed wprowadzeniem wartości z kapelusza :)